server.py

```diff
@@ -137,6 +144,8 @@ def get_user_from_request(request, dao):
 
 
 def is_user_admin_for_region(user, region):
+    if user.admin_level == 'SUPER':
```

Comment:
if we want to stick with the Enum method, these 'SUPER's and everything should be M.AdminLevels.SUPERs.